doc/api/packages.md

```diff
@@ -485,7 +485,7 @@ where `import '#dep'` does not get the resolution of the external package
 file `./dep-polyfill.js` relative to the package in other environments.
 
 Unlike the `"exports"` field, the `"imports"` field permits mapping to external
-packages.
+packages and locations.
```

Comment:
Does https://nodejs.org/api/esm.html#resolution-algorithm-specification need to be updated as well?